src/components/Moviepage/MoviePage.jsx

```diff
@@ -1,5 +1,33 @@
 import React from "react";
 
 export default function MoviePage({ movieInfo }) {
-  return <div>{movieInfo.id}</div>;
+  return (
+    <div>
+      <p className="pt-10 pb-4 text-5xl">{movieInfo.title}</p>
+      <div className="flex p-10">
+        <img
+          className="rounded ml-4"
+          src={`http://image.tmdb.org/t/p/w400/${movieInfo.poster_path}`}
+        />
+        <div className="text-left px-10">
+          <p className="py-2 text-lg">{movieInfo.overview}</p>
+          <div className="mt-10">
+            <p className="py-2">
+              Genres:
+              {movieInfo?.genres?.map((genre, index) => (
+                <span key={index} className="pl-2">
+                  {genre.name}
+                </span>
+              ))}
+            </p>
+
```

Comment:
I can see flickering. To reproduce the bug:
-open homepage and click on any movie;
-return to the homepage and click on some another movie.
Result: You will see the old poster image for a decimal of a second.
Expected result: loading spinner. 

Please fix this issue.

Reply:
fixed